cicd/docker/algod/setup.py

```diff
@@ -57,17 +66,7 @@ def algod_directories(network_dir):
 
     return data_dir, kmd_dir
 
-
-def create_network(bin_dir, network_dir, template, token, algod_port, kmd_port):
-    """
-    Create a private network.
-    """
-    # Reset network dir before creating a new one.
-    if os.path.exists(args.network_dir):
-        shutil.rmtree(args.network_dir)
-
-    # $BIN_DIR/goal network create -n testnetwork -r $NETWORK_DIR -t network_config/$TEMPLATE
-    subprocess.check_call(['%s/goal network create -n testnetwork -r %s -t %s' % (bin_dir, network_dir, template)], shell=True)
+def configure_network(network_dir, token, algod_port, kmd_port):
```

Comment:
Why do we need to split the create and configure?

If you want it to be more general purpose, make the port/token optional fields and don't set them when they aren't provided.

Reply:
That's kind of the rationale that led me to split it up this way. If we want to be able to set ports/tokens at any point, we would need a script that edits the respective config files to make that happen. Is there another way we could do that?

Reply:
I think the docker way to do this would be to hard code the port in the container, and map it to a different host port when the container starts.

To update the tokens you have to update the token files then restart algod/kmd.